build/rewrite.go

```diff
@@ -237,7 +294,9 @@ func fixLabels(f *File) {
 					continue
 				}
 				key, ok := as.LHS.(*Ident)
-				if !ok || !tables.IsLabelArg[key.Name] || tables.LabelDenylist[callName(v)+"."+key.Name] {
+				var isLabelArg map[string]bool = *w.LabelArg
```

Comment:
So this would become something like:
```
var isLabelArg map[string]bool = tables.IsLabelArg
if w.LabelArg != nil {
  isLabelArg = w.LabelArg
}
```